frappe/website/doctype/blog_post/templates/blog_post_row.html

```diff
@@ -1,7 +1,10 @@
 {%- set post = doc -%}
 <div class="web-list-item blog-list-item">
 	<div class="row">
-		<div class="col-xs-12">
+		<div class="col-xs-6">
```

Comment:
There should be if here, if there is no image, existing layouts should work as it is
`{% if blog_image %}`